[2025-09-06T09:55:52.233-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-06T09:55:52.252-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_FLIGHTS.validate manual__2025-09-06T12:54:32.083155+00:00 [queued]>
[2025-09-06T09:55:52.258-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_FLIGHTS.validate manual__2025-09-06T12:54:32.083155+00:00 [queued]>
[2025-09-06T09:55:52.259-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-06T09:55:52.712-0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): validate> on 2025-09-06 12:54:32.083155+00:00
[2025-09-06T09:55:52.728-0300] {standard_task_runner.py:63} INFO - Started process 55361 to run task
[2025-09-06T09:55:52.740-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ETL_FLIGHTS', 'validate', 'manual__2025-09-06T12:54:32.083155+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/ETL_FLIGHTS.py', '--cfg-path', '/tmp/tmpp8f26w94']
[2025-09-06T09:55:52.745-0300] {standard_task_runner.py:91} INFO - Job 17: Subtask validate
[2025-09-06T09:55:53.285-0300] {task_command.py:426} INFO - Running <TaskInstance: ETL_FLIGHTS.validate manual__2025-09-06T12:54:32.083155+00:00 [running]> on host sebastian-Aspire-A515-54
[2025-09-06T09:55:54.653-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ETL_FLIGHTS' AIRFLOW_CTX_TASK_ID='validate' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T12:54:32.083155+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T12:54:32.083155+00:00'
[2025-09-06T09:55:54.658-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-06T09:55:54.686-0300] {ETL_FLIGHTS.py:90} INFO - validate: OK, inserted 100 rows
[2025-09-06T09:55:54.687-0300] {python.py:237} INFO - Done. Returned value was: None
[2025-09-06T09:55:54.688-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-06T09:55:54.703-0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_FLIGHTS, task_id=validate, run_id=manual__2025-09-06T12:54:32.083155+00:00, execution_date=20250906T125432, start_date=20250906T125552, end_date=20250906T125554
[2025-09-06T09:55:56.532-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-06T09:55:56.567-0300] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-06T09:55:56.568-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
